### DISCUSSION TITLE
How to modify /etc/prometheus/prometheus.yml?

### DISCUSSION THREAD

**Discussion by hillbun** [prometheus-operator/kube-prometheus#986] [Q&A]
How to modify /etc/prometheus/prometheus.yml?

I can not find any configmap for it

**Comment by hillbun** [comment]
I need to add targets, thanks.

**Comment by paulfantom** [comment]
In short - you don't as this is prometheus-operator job.

Adding targets can be done by creating [ServiceMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#servicemonitor), [PodMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#podmonitor), or [Probe](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#probe) objects. We do have a short getting started guide at https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md
